common/views/components/Pagination/Pagination.tsx

```diff
@@ -124,7 +124,8 @@ export const Pagination: FunctionComponent<Props> = ({
       )}
 
       <span>
-        Page <strong>{currentPage}</strong> of {formatNumber(totalPages)}
+        Page <strong>{currentPage}</strong> of{' '}
+        {formatNumber({ number: totalPages })}
```

Comment:
So this is one of the problems of keyword args; it makes calls where you don't need the disambiguity more verbose.

In these cases I like to have a mix of positional/keyword arguments, so you could do:

```javascript
formatNumber(42)
formatNumber(42, { compact: true })
formatNumber(42, { compact: false })
```

which you get by making the second parameter an optional object.

IMO this gets you a good blend of concision and readability, see e.g. https://github.com/wellcomecollection/wellcomecollection.org/blob/main/prismic-model/src/parts/boolean.ts

Reply:
Looks like the best solution, adding a name for the number really wasn't my fave. I'll make the change.